operators/config/e2e/global_operator.yaml

```diff
@@ -0,0 +1,219 @@
+---
```

Comment:
I wanted to avoid the duplication too. The problem is that if I add templating to the manifests in the `config/operator` directory, then users will not be able to easily run them from the Makefile provided in that directory. They will have to have Go installed to render the templates -- which is not nice. The other issue is that I need to add test-specific things to the manifests such as labels which are not strictly necessary for a casual user trying out the operator. I felt that adding those to user-facing examples would make it seem too complicated.

I am open to any suggestions that could help avoid the duplication.

Reply:
I don't have anything better to propose :/
Fine with going with the approach you propose here.